lib/firmata.js

```diff
@@ -209,20 +211,28 @@ SYSEX_RESPONSE[CAPABILITY_RESPONSE] = function(board) {
 
   // Only create pins if none have been previously created on the instance.
   if (!board.pins.length) {
+    var pin = {};
     for (var i = 2, n = 0; i < board.currentBuffer.length - 1; i++) {
       if (board.currentBuffer[i] === 127) {
```

Comment:
Add the following comment above this line:
`// 127 denotes the end of the pin's modes`